clients/cobol-lsp-vscode-extension/src/constants.ts

```diff
@@ -27,6 +27,7 @@ export const SETTINGS_TAB_CONFIG: string = "cobol-lsp.smart-tab";
 
 export const SERVER_PORT = "cobol-lsp.server.port";
 export const SERVER_RUNTIME = "cobol-lsp.serverRuntime";
+export const DIALECT_LIBS = "cobol-lsp.dialect.libs";
```

Comment:
This is a new request that the server asks the client. This request also takes dialect as a parameter. The client handles this request in a similar manner as for copybooks for consistency across the project. It looks into the settings.json as well as proc_grps.